erc/ERCS/erc-7562.md

```diff
@@ -299,41 +394,41 @@ These attacks are examples provided to describe and rationalise the reason for t
 * Total attack cost
     * Polygon (@150 gwei) - $18 + $1.5 per block ($2700/hour)
     * Mainnet: (@40 gwei) - $51840 + $5040 per block ($1.5M/hour)
-    * Note that the above costs ignore the gas-price increase because the underlying network is also overloaded.
+    * Note that the above costs ignore the gas price increase because the underlying network is also overloaded.
 * Mitigations
-    * A **staked factory:** 
+    * A **staked factory:**
         * Can attempt such an attack once, since after invalidating a single bundle creation its reputation drops and another staked factory has to be used.
         * So a successful attack needs to deploy and stake a new paymaster on every block. The old paymaster's stake is locked and can be re-used only after 24 hours.
     * For **unstaked factory**:
         * As per **[UREP-020]**, it is initially allowed only 10 `UserOperations` in the mempool.
-        * That value increases fast, but drops (to negative) on invalidations.
+        * That value increases fast but drops (to negative) on invalidations.
         * It needs to successfully deploy 10 `UserOperations` to get a reputation to deploy a batch of 1000 (which attack).
```

Comment:
I don't understand this sentence.  I assume it needs to successfully deploy 10 **accounts** rather than **UserOperations**, but I don't understand the **(which attack)** part.

In general, if I didn't know this attack, I doubt I'd be able to understand the attack and its high costs from just reading this section.  Is it really valuable to have this example in the ERC?  Especially highlighting it as the first item, before even discussing attacks fully mitigated by the rules?